app/Http/Controllers/V2/Dashboard/TotalTerrafundHeaderDashboardController.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+namespace App\Http\Controllers\V2\Dashboard;
+
+use App\Http\Controllers\Controller;
+use App\Models\V2\Projects\Project;
+use App\Models\V2\Projects\ProjectReport;
+use App\Models\V2\Sites\SiteReport;
+use App\Models\V2\TreeSpecies\TreeSpecies;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\DB;
+
+class TotalTerrafundHeaderDashboardController extends Controller
+{
+    public function __invoke(Request $request)
+    {
+        return response()->json([
+            'total_non_profit_count' => $this->getTotalNonProfitCount($request),
+            'total_enterprise_count' => $this->getTotalEnterpriseCount($request),
+            'total_entries' => $this->getTotalJobsCreatedSum($request),
+            'total_hectares_retored' => $this->getTotalHectaresRestoredGoalSum($request),
+            'total_trees_restored' => $this->getTotalTreesRestoredSum($request),
+            'total_trees_restored_goal' => $this->getTotalTreesGrownGoalSum($request),
+        ]);
+    }
+
+    public function buildQueryFromRequest($query, $request)
```

Comment:
still missing the `where('framework_key', 'terrafund')` that is repeated in all calls right?

Reply:
and the method should go into another file as a helper, so we can import it in other endpoints.